contentcuration/contentcuration/static/js/edit_channel/sharedComponents/client.js

```diff
@@ -0,0 +1,66 @@
+import axios from 'axios';
```

Comment:
Why has this been added here, rather than using the already existing shared client module?

Reply:
Ah this was before your frontend PR got merged. Will consolidate!